hw3/Task_13.ts

```diff
@@ -0,0 +1,20 @@
+//Task 13
+let goTo: string = 'восток';
+
+switch(goTo) {
+case 'юг':
```

Comment:
Please format you code, at least add tabs if you loops.
In webstorm formating is:
<img width="294" alt="image" src="https://user-images.githubusercontent.com/47811445/215258133-fb1fe805-911b-4b97-a953-c85231df05d2.png">